bosh-setup/README.md

```diff
@@ -4,11 +4,13 @@
     <img src="http://azuredeploy.net/deploybutton.png"/>
```

Comment:
Sure, if you're taking over the ownership of the template. btw is qingfuwang on your team as well?